src/avfoundation.cs

```diff
@@ -13172,52 +13407,48 @@ interface AVContentKeySession_AVContentKeyRecipients
 		IAVContentKeyRecipient[] GetContentKeyRecipients ();
 	}
 
-#if WATCH
-	[Static]
-#endif
 	[TV (10,2), Mac (10,12,4), iOS (10,3), Watch (6,0)]
 	[DisableDefaultCtor]
 	[BaseType (typeof (NSObject))]
 	interface AVContentKeyRequest {
-		[NoWatch]
+		[Watch (7,0)]
 		[Field ("AVContentKeyRequestProtocolVersionsKey")]
 		NSString ProtocolVersions { get; }
 
-		[NoWatch]
+		[Watch (7,0)]
 		[Export ("status")]
 		AVContentKeyRequestStatus Status { get; }
 
-		[NoWatch]
+		[Watch (7, 0)]
 		[NullAllowed, Export ("error")]
 		NSError Error { get; }
 
-		[NoWatch]
+		[Watch (7, 0)]
 		[NullAllowed, Export ("identifier")]
 		NSObject Identifier { get; }
```

Comment:
Is it possible to do any better than an `NSObject` for `Identifier`?

Header:

```
/*
 @property      identifier
 @abstract      Container- and protocol-specific identifier for the content key.
 @discussion    In order to use a key with an HTTP Live Streaming AVURLAsset, the identifier must be an NSURL that matches a key URI in the Media Playlist.
*/
@property (readonly, nullable) id identifier;
```

Reply:
Mope, identifiers are just an NSObject, we do not care about it. It is used to later retrieve it. This is 'common' in other APIs.